src/components/atoms/CheckToDisplayAsFull/CheckToDisplayAsFull.tsx

```diff
@@ -27,34 +29,20 @@ const CheckToDisplayAsFull = ({
     return 'Full';
   }, [spaceLeft, capacity]);
 
-  const spaceAvailable = spaceLeft && spaceLeft < 0;
+  const spaceAvailable = spaceLeft === 0;
 
   return (
-    <>
-      {spaceAvailable ? (
-        <View justifyContent={'space-between'} flexDirection={'row'}>
-          <Text
-            fontFamily={'body'}
-            fontWeight="500"
-            fontSize={14}
-            color={subheadingTextColorFull}>
-            Full
-          </Text>
-          <ReservedCount count={`${-spaceLeft}`} />
-        </View>
-      ) : (
-        <Text
-          fontFamily={'body'}
-          fontWeight="400"
-          fontSize={14}
-          color={
-            spaceLeft === 0 ? subheadingTextColorFull : subheadingTextColor
-          }
-          testID={`DayTimeSelectorSubheading-${id}`}>
-          {subheadingText}
-        </Text>
-      )}
-    </>
+    <View justifyContent={'space-between'} flexDirection={'row'}>
+      <Text
+        fontFamily={'body'}
+        fontWeight={spaceAvailable ? '500' : '400'}
+        fontSize={14}
+        color={spaceLeft === 0 ? subheadingTextColorFull : subheadingTextColor}
```

Comment:
`spaceLeft === 0` can be replace with `spaceAvailable`?

Reply:
Will amend